src/components/layout.tsx

```diff
@@ -12,7 +12,7 @@ interface LayoutProps {
 
 function Layout ({
   contentContainerClass = 'content-fullscreen-tablet',
-  rootContainerClass = 'root-container-default',
+  rootContainerClass = 'root-container-default flex flex-col',
```

Comment:
how about add `flex flex-col` to `root-container-default` class in 
https://github.com/OpenBeta/open-tacos/blob/user-page-footer/src/styles/defaults.css#L16